Makefile

```diff
@@ -1,27 +1,21 @@
-CORAL_DIST_PATH = coral/dist
+version = 1.1.0
 
-default:
-	@echo "TODO: ADD USAGE INSTRUCTIONS"
+build_all: klaw_core cluster_api
 
-$(CORAL_DIST_PATH): coral/index.html coral/src/*
-	pnpm  --prefix coral build --assetsDir assets/coral --mode springboot
+klaw_core:
+	cd core && mvn clean verify
 
-src/main/resources/templates/coral/index.html: $(CORAL_DIST_PATH)
-	mkdir -p $(shell dirname $@)
-	cp -r $(CORAL_DIST_PATH)/index.html $@
+cluster_api:
+	cd cluster-api && mvn clean verify
 
-src/main/resources/static/assets/coral: $(CORAL_DIST_PATH)
-	cp -r $(CORAL_DIST_PATH)/assets/* $@
+edit-core-config:
+	${EDITOR} core/target/classes/application.properties
```

Comment:
i fear it could be a hidden conflict because of several independent changes
or am I wrong?

Reply:
Makefile of UI is moved to /coral. For now, there would not be any conflicts